include/spirv-tools/optimizer.hpp

```diff
@@ -230,13 +230,14 @@ Optimizer::PassToken CreateNullPass();
 // Section 3.32.2 of the SPIR-V spec) of the SPIR-V module to be optimized.
 Optimizer::PassToken CreateStripDebugInfoPass();
 
-// Creates a strip-reflect-info pass.
-// A strip-reflect-info pass removes all reflections instructions.
-// For now, this is limited to removing decorations defined in
-// SPV_GOOGLE_hlsl_functionality1.  The coverage may expand in
-// the future.
+// [Deprecated] This will not create a strip-nonsemantic-info pass.  See below.
```

Comment:
Do you mean "This will create a "....  (drop the "not") ?

Reply:
I do this typo way too often.  I intended to write "now".  However, that is not needed either.  Fixed.